### ISSUE TITLE
Can't get 2nd connection to room working

### ISSUE THREAD

**Issue by petervnv** [erbbysam/webRTCCopy#19]
Hi,

I'm experimenting with your software. Thanks for providing it, by the way. Very cool.

However, I'm having trouble getting a second client to connect to a room.
With client1 I create the room and user normally, with otr disabled.
Then the 2nd client types in the same URL, is prompted for a username, and then it shows the room (although it seems empty - ie client 1 is not there). At the bottom, it shows "Please wait a few seconds to connect over webRTC."
Client 1 is still in the room, although it also shows empty for him.

I have tried client2 from different wifi and 3g networks, thinking that maybe this was a NAT or FW problem. Unfortunately client2 is never able to connect and so file transfer and IM can't happen.

Since that didn't work on your site, I installed it on my own server in the cloud. Unfortunately I'm seeing the exact same behavior. Looking at my nginx logs, I don't see any errors.

Any clues?

Thanks

**Comment by petervnv** [comment]
Ooops, my mistake.
server.js had crashed on my site.

It seems to be working now.
Also, do you have turn support? I didn't see it anywhere.

**Comment by erbbysam** [comment]
Nope, no TURN support. Didn't want to potentially have to support such a large amount of traffic :) That said, I could have probably built it in, then disabled it... I really have no idea how hard TURN support is to add now that I think about it more.

**Comment by petervnv** [comment]
Hi,
Thanks for replying.

Taking a look at the code, I wonder if something as simple as this would suffice to enable STUN and ICE? I'm no webrtc expert, but from I read it may work.
What do you think?

 var rtc.SERVER == {
  'iceServers': [
    {
      'url': 'stun:stun.l.google.com:19302'
    },
    {
      'url': 'turn:turnserver1:3478?transport=tcp',
      'credential': 'credential,
      'username': 'username'
    }
  ]
}
